stacks_queues/stack.py

```diff
@@ -21,18 +25,25 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.length == 0:
+            raise StackEmptyException ("stack is empty")
+
+        return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        self.store.length = 0
+        if self.store.length == 0:
+            return True
+        return False
```

Comment:
🤔 I'm not sure this works... it will always return `True` since you set the length to 0 on line 37.
```suggestion
        if self.store.length == 0:
            return True
        return False
```

Reply:
Good catch, thanks!